website/docs/guides/docs/docs-create-doc.mdx

```diff
@@ -185,6 +210,41 @@ slug: /
 Lorem ipsum
 ```
 
+#### Advanced: Overriding Slugs Programmatically
```

Comment:
The goal of this page is to be accessible and lightweight for newcomers. The goal is to cover greenfield projects starting from our init template, rather than migrations from one docs system to another (can could have another page dedicated to that topic, if you want to work on it)

`parseFrontMatter` is an advanced feature for which there's an infinite number of use-cases, and we can't document them all in our guides. 

Note that the section you wrote could apply to the page and blog plugins as well, not just docs.

I'd prefer to not have this section in our "create a doc" guide